wrappers/dotnet/indy-sdk-dotnet-test/AgentTests/AgentAddIdentityTest.cs

```diff
@@ -1,8 +1,9 @@
-﻿using Indy.Sdk.Dotnet.Wrapper;
+﻿using Hyperledger.Indy.Sdk.AgentApi;
+using Hyperledger.Indy.Sdk.SignUsApi;
 using Microsoft.VisualStudio.TestTools.UnitTesting;
 using System.Threading.Tasks;
 
-namespace Indy.Sdk.Dotnet.Test.Wrapper.AgentTests
+namespace Hyperledger.Indy.Sdk.Test.AgentTests
```

Comment:
Can we remove `.Sdk.` part of packages names? In our current vision SDK is complex of wrappers, lib, docs, samples etc. So, in wrapper it seems like redundant.

Reply:
Sounds reasonable, so sure.